custom/icds_reports/utils/aggregation_helpers/daily_attendance.py

```diff
@@ -59,7 +59,8 @@ def aggregate_query(self):
                 pse_conducted as pse_conducted,
                 supervisor_id as supervisor_id
               FROM "{ucr_daily_attendance_tablename}"
-              WHERE month = %(start_month)s
+              WHERE month = %(start_month)s and
+                    awc_open_count+awc_not_open = 1
```

Comment:
Hey, I have made the change to only include the required forms.
One questions. I know that there will be condition where either of the question is answered yes to be considered yes. will there a condition where  one question is answered no and other is blank? HERE:https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/ucr/data_sources/daily_feeding_forms.json#L729-L759

If yes, Current modification will ingore that form.
@calellowitz

Reply:
booleans should coerce it to 0 or 1, so this should work. However I think `(awc_open_count = 1 OR awc_not_open = 1)` is clearer

Reply:
> Hey, I have made the change to only include the required forms.
> One questions. I know that there will be condition where either of the question is answered yes to be considered yes. will there a condition where one question is answered no and other is blank? HERE:https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/ucr/data_sources/daily_feeding_forms.json#L729-L759
> 
> If yes, Current modification will ingore that form.
> @calellowitz

Checked with Mruga, It wont be possible so we are good.

Reply:
Agree with emord's statement about clarity (generally not a huge fan of doing arithmetic on booleans like this since i think the intent is less clear and it is barely more concise)